src/components/BlockNews/BlockNews.vue

```diff
@@ -1,17 +1,20 @@
 <template>
-    <nuxt-link class="block-news" :to="to">
+    <nuxt-link :class="classes" :to="to">
         <wp-image class="block-image" :image="image">
             <span class="more">More</span>
         </wp-image>
         <div class="block-text">
-            <time class="date" v-html="formattedDate" />
-            <h3 class="title" v-html="title" />
-            <p class="excerpt" v-html="stripTags(excerpt)" />
-            <div v-if="categories" class="categories">
-                <span
-                    v-for="(category, i) in categories"
-                    v-html="category.name + comma(i)"
-                />
+            <p class="prompt">Next</p>
```

Comment:
Don't use a `<p>` for this, it's not a paragraph of text. `span` is fine. Also, add a `v-if` on it.

Reply:
@drewbaker Yep, there was a discrepancy in the XD file, but I confirmed with Rick and you're right about the "next" placement, excerpt